core/drivers/imx_wdog.c

```diff
@@ -154,18 +154,16 @@ static TEE_Result imx_wdog_base(vaddr_t *wdog_vbase)
 	*wdog_vbase = (vaddr_t)phys_to_virt(WDOG_BASE, MEM_AREA_IO_SEC, 1);
 #if defined(CFG_IMX_WDOG_EXT_RESET)
 	ext_reset_output = true;
+#else
+	ext_reset_output = false;
```

Comment:
Global variables are always initialized to `false` by default, so this is not desirable. I'd much prefer you add `__maybe_unused` to the definition of `ext_reset_output`.